specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+{
+   "swagger": "2.0",
+   "info": {
+      "title": "Cognitive Services API",
+      "description": "Cognitive Services API to support OpenAI Dall-E",
+      "version": "2023-04-01-preview"
+   },
+   "host": "eastus.api.cognitive.microsoft.com",
+   "basePath": "/dalle",
+   "schemes": ["https"],
+   "securityDefinitions": {
+      "Api-Key": {
+        "type": "apiKey",
+        "name": "Api-Key",
+        "in": "header",
+        "description": "Provide your Cognitive Services Azure OpenAI account key here."
+      }
+   },
+   "security": [
+      {
+        "Api-Key": []
+      }
+    ],
```

Comment:
"securityDefinitions": {
    "api-key": {
      "type": "apiKey",
      "name": "api-key",
      "in": "header",
      "description": "Provide your Cognitive Services Azure OpenAI account key here."
    }
  },
  "security": [
    {
      "api-key": []
    }
  ],
  "schemes": [
    "https"
  ],
  "x-ms-parameterized-host": {
    "hostTemplate": "{endpoint}/openai",
    "useSchemePrefix": false,
    "parameters": [
      {
        "$ref": "#/parameters/endpoint"
      }
    ]
  }


and move to bottom of doc

Reply:
Fixed